projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -509,19 +509,21 @@ const VideoPressEdit = CoreVideoEdit =>
 			const isFetchingVideo = isFetchingMedia || isFetchingPreview;
 			const isVideoFallbackOrNotPreview = fallback || ! preview;
 			// If the video is uploading or fetching, we should display the Loading component.
-			const displayLoadingBlock = isUploading || isFetchingVideo;
+			// Once we have a preview, we'll bypass the following return to make sure we can keep blockSettings rendered.
+			const displayLoadingBlock = ( isUploading || isFetchingVideo ) && ! preview;
```

Comment:
Since we now have 2 loading blocks, I wonder if this should be more specific? 
```suggestion
			const displayMainLoadingBlock = ( isUploading || isFetchingVideo ) && ! preview;
```

Reply:
I changed it to "displayNoPreviewLoadingBlock", while still not being perfect, I think it's a bit more descriptive than "main" and matches what's in the comment over it :)